src/generated/api/src/model.rs

```diff
@@ -3098,8 +3098,8 @@ pub struct Page {
     #[serde(skip_serializing_if = "std::string::String::is_empty")]
     pub name: std::string::String,
 
-    /// The Markdown content of the page. You can use <code>&#40;== include {path}
-    /// ==&#41;</code> to include content from a Markdown file. The content can be
+    /// The Markdown content of the page. You can use \<code\>&#40;== include {path}
```

Comment:
This change is good, but we should also fix this upstream: it should be using a code span (in markdown), instead of `<code> ... </code>`.

Reply:
Makes sense, will up a change for this.